_data/navigation.yml

```diff
@@ -164,7 +164,7 @@ guides:
             url: /guides/tutorials/codelabs/hello-deployment/
           - title: GCE Source To Prod
             url: /guides/tutorials/codelabs/gce-source-to-prod/
-          - title: Kubernete Source To Pro
+          - title: Kubernete Source To Prod
```

Comment:
If you're going to put the 'd' in 'Prod', might as well put the 's' in 'Kubernetes'.